community/perl-anyevent-i3/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=perl-anyevent-i3
+_pkgname=AnyEvent-I3
+pkgver=0.17
+pkgrel=0
+pkgdesc="Communicate with the i3 window manager."
+url="https://metacpan.org/release/AnyEvent-I3"
+arch="noarch"
+license="Artistic-1.0 GPL-1.0-only"
+cpandepends="perl-anyevent perl-json-xs"
```

Comment:
All non-official variables should start with an `_`. See the section about [variables][0] in APKBUILD reference

[0]:https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#Variables

Reply:
I know it. So every perl module generated by apkbuild-cpan (which seems most of them) has to to obey this rule.